integration-tests/src/test/java/tachyon/hadoop/TFSAclIntegrationTest.java

```diff
@@ -0,0 +1,174 @@
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.hadoop;
+
+import java.io.IOException;
+import java.net.URI;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FSDataOutputStream;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.fs.permission.FsPermission;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import tachyon.Constants;
+import tachyon.LocalTachyonClusterResource;
+import tachyon.security.authentication.AuthType;
+
+/**
+ * Integration tests for {@link TFS#setOwner(Path, String, String)} and
+ * {@link TFS#setPermission(Path, org.apache.hadoop.fs.permission.FsPermission)}.
+ */
+public final class TFSAclIntegrationTest {
+
+  private static final int BLOCK_SIZE = 1024;
+  @ClassRule
+  public static LocalTachyonClusterResource sLocalTachyonClusterResource =
+      new LocalTachyonClusterResource(100000000, 100000, BLOCK_SIZE,
+          Constants.SECURITY_AUTHENTICATION_TYPE, AuthType.SIMPLE.getAuthName(),
+          Constants.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, "true");
+  private static FileSystem sTFS;
+
+  private static void create(FileSystem fs, Path path) throws IOException {
+    FSDataOutputStream o = fs.create(path);
+    o.writeBytes("Test Bytes");
+    o.close();
+  }
+
+  public static void cleanup(FileSystem fs) throws IOException {
+    FileStatus[] statuses = fs.listStatus(new Path("/"));
+    for (FileStatus f : statuses) {
+      fs.delete(f.getPath(), true);
+    }
+  }
+
+  @BeforeClass
+  public static void beforeClass() throws Exception {
+    Configuration conf = new Configuration();
+    conf.set("fs.tachyon.impl", TFS.class.getName());
+
+    URI uri = URI.create(sLocalTachyonClusterResource.get().getMasterUri());
+
+    sTFS = FileSystem.get(uri, conf);
+  }
+
+  @After
+  public void cleanupTFS() throws Exception {
+    cleanup(sTFS);
+  }
+
+  /**
+   * Test for {@link TFS#setPermission(Path, org.apache.hadoop.fs.permission.FsPermission)}. It
+   * will test changing the permission of file using TFS.
+   */
+  @Test
+  public void chmodTest() throws Exception {
+    Path fileA = new Path("/chmodfileA");
+
+    create(sTFS, fileA);
+    FileStatus fs = sTFS.getFileStatus(fileA);
+    short permission = fs.getPermission().toShort();
```

Comment:
nit: no need for a variable if the result is the result is only used in the assert (similar to the 0755 assert).